vscode-dotnet-runtime-library/src/Acquisition/IDistroDotnetSDKProvider.ts

```diff
@@ -81,7 +81,8 @@ export abstract class IDistroDotnetSDKProvider {
         this.distroJson = JSON.parse(fs.readFileSync(distroDataFile, 'utf8'));
         if(!distroVersion || !this.distroJson || !((this.distroJson as any)[this.distroVersion.distro]))
         {
-            const error = new DotnetAcquisitionDistroUnknownError(new Error('We are unable to detect the distro or version of your machine'),
+            const error = new DotnetAcquisitionDistroUnknownError(new Error(`Automated installation for the distro ${this.distroVersion.distro} is not yet supported.
+Please install the .NET SDK manually: dotnet.microsoft.com/download.`),
```

Comment:
I think you should leave off the `.` on the end so copying the download link is easy. Also, the link should be a full link, meaning `https://dotnet.microsoft.com/download`.

Reply:
Good point, thank you